Source/Widgets/Viewer/Viewer.js

```diff
@@ -438,6 +438,7 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
 
         var sceneModePicker;
         if (!scene3DOnly && (!defined(options.sceneModePicker) || options.sceneModePicker !== false)) {
+            eventHelper.add(cesiumWidget.scene.morphStart, Viewer.prototype._clearTrackedObject, this);
```

Comment:
Even if there is no scene mode widget, scene transitions can still happen, so I would move this line outside of the if block.  Once day we'll fix the stupid zoom to home transition behavior.